### PR TITLE
Revert "Adding Pewaukee to master"

### PR DESCRIPTION
**Pull request by dozinbatty**
Reverts githubteacher/everyone-july#131

### CODE REVIEW

File: places/dozinbatty-pewaukee.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-## Pewaukee, WI
```
Comment (ae14) [inline places/dozinbatty-pewaukee.md]: What's the weather like in Pewaukee?

**Comment by vicuk** [comment]
Where does the name Pewaukee come from @dozinbatty ?

**Comment by jaydave116** [comment]
Nice work :smile:

**Comment by jaydave116** [comment]
Good to go. :ship:

**Pull request merged**